app/models/school.rb

```diff
@@ -94,4 +102,23 @@ def organisation_type
   def primary_or_secondary_only?
     [PRIMARY_PHASE, SECONDARY_PHASE].include?(phase)
   end
+
+  def address
+    [
+      address1,
+      address2,
+      address3,
+      town,
+      postcode,
+      "United Kingdom",
+    ].compact.join(", ")
+  end
+
+  private
+
+  def address_changed?
+    address1_changed? || address2_changed? ||
+      address3_changed? || town_changed? ||
+      postcode_changed?
```

Comment:
```suggestion
  (changed & %w[address1 address2 address3 town postcode]).any?
```

Not saying this is better, may be more of a cognitive load but another way you could tackle it?

Reply:
resolved in [c33d97b](https://github.com/DFE-Digital/itt-mentor-services/pull/382/commits/c33d97b1f77a5bbdd5a6212b8e5e6bff47a6dd8d)